frontend/src/constants/MapboxConstants.js

```diff
@@ -0,0 +1,2 @@
+export const MapboxAccessToken =
+  'pk.eyJ1IjoiY3Jvd2VhdHgiLCJhIjoiY2o1NDFvYmxkMHhkcDMycDF2a3pseDFpZiJ9.UcnizcFDleMpv5Vbv8Rngw';
```

Comment:
We should probably think about how to handle token security.

I was considering suggesting moving it to an environment variable but it'll still be easily accessible via view source that way.

It looks like there's a pretty cool [tokens API](https://www.mapbox.com/api-documentation/#tokens) available that we could use to generate temporary tokens. Definitely out of scope for this PR but worth investigating.

Reply:
This is a public token. There isn't a good way to keep a token like this secret, since we need it when the page runs in the users browser.